platform/ios/CHANGELOG.md

```diff
@@ -154,7 +154,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ## 3.5.2 - April 7, 2017
 
 * Fixed an issue that caused a crash when the user location annotation was presenting a callout view and the map was moved. ([#8686](https://github.com/mapbox/mapbox-gl-native/pull/8686))
-* This release was built with Xcode 8.3.1, which fixed [a significant bitcode issue](http://www.openradar.me/31302382) introduced in Xcode 8.3 that caused Mapbox iOS SDK 3.5.1 to be 2× larger than 3.5.0.
+* This release was built with Xcode 8.3.1, which fixed [a significant bitcode issue](http://www.openradar.me/31302382) introduced in Xcode 8.3 that caused Mapbox Maps SDK for iOS 3.5.1 to be 2× larger than 3.5.0.
```

Comment:
No need to rename references to previous versions of the SDK that were in fact named “Mapbox iOS SDK”. But while you’re here, can you add a blurb to the top of the changelog about the renaming? (Same for the macOS map SDK changelog.)